pkg/crypto/tinkcrypto/primitive/composite/ecdhes/subtle/ecdhes_aead_composite_common.go

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package subtle
```

Comment:
why is this package called subtle?

Reply:
(subtle seems like a container folder rather than a package)

Reply:
Just like Tink's `subtle` packages that contains the core crypto logic for each primitive, they're used by the outer layer of crypto primitives and key templates which are exposed to the API user. 

I'm following their pattern here.

Reply:
So the functions in this subtle package are not intended to be used except by its child packages?

Reply:
except by its parent package, `ecdhes`

But the jwe package will reference the common fields to unmarshal the ciphertext container data.